dev-libs/gtest/gtest-1.14.0.recipe

```diff
@@ -59,10 +59,10 @@ BUILD_PREREQUIRES="
 	"
 
 defineDebugInfoPackage gtest$secondaryArchSuffix \
-	$libDir/libgmock.so.1.11.0 \
-	$libDir/libgmock_main.so.1.11.0 \
-	$libDir/libgtest.so.1.11.0 \
-	$libDir/libgtest_main.so.1.11.0
+	$libDir/libgmock.so.1.14.0 \
+	$libDir/libgmock_main.so.1.14.0 \
+	$libDir/libgtest.so.1.14.0 \
+	$libDir/libgtest_main.so.1.14.0
```

Comment:
Not checked, but maybe libVersion(Compat) can be used here?

Reply:
Will change if can be used after someone actually checks... I'm not familiar enough yet...

Reply:
If people would want to edit these lines on every upgrade, it might be better to use `$portVersion` instead of hard-coding the version... Google Test doesn't guarantee compatibility between two builds or two versions.

Reply:
libVersions isn't always the same as the portVersion :)

Reply:
> libVersions isn't always the same as the portVersion :)

I know... One good example is zlib-ng in compat mode...